pkg/server/routes.go

```diff
@@ -44,12 +43,7 @@ import (
 	"github.com/cms-enterprise/easi-app/pkg/upload"
 )
 
-func (s *Server) routes(
-	contextMiddleware func(handler http.Handler) http.Handler,
-	corsMiddleware func(handler http.Handler) http.Handler,
-	traceMiddleware func(handler http.Handler) http.Handler,
-	loggerMiddleware func(handler http.Handler) http.Handler,
```

Comment:
was just something i was trying to pass fewer functions around - not married to it